VAMobile/src/screens/BenefitsScreen/ClaimsScreen/AppealDetailsScreen/AppealStatus/AppealCurrentStatus/AppealCurrentStatus.test.tsx

```diff
@@ -434,14 +434,14 @@ context('AppealStatus', () => {
           "Your appeal was merged with another appeal. The Board of Veterans'  Appeals merges appeals so that you can receive a single decision on as many appealed issues as possible. This appeal was merged with an older appeal that was closest to receiving a Board decision.",
         ),
       ).toBeTruthy()
-      expect(screen.getByText('Check')).toBeTruthy()
-      expect(screen.getByText('Your claims and appeals')).toBeTruthy()
+      expect(screen.getByText('Check your ')).toBeTruthy()
```

Comment:
Our [best practice](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/FrontEnd/Testing/UnitTesting#targeting-by-translated-text) is to use the translation strings instead of direct text